ray-operator/controllers/ray/batchscheduler/kai-scheduler/kai_scheduler.go

```diff
@@ -0,0 +1,66 @@
+package kaischeduler
+
+// This KAI plugin relies on KAI-Scheduler's
```

Comment:
Cool! In that case, I guess it’s possible for the KAI scheduler to support gang scheduling with autoscaling enabled?